closed/src/java.base/share/classes/sun/security/ec/NativeECDHKeyAgreement.java

```diff
@@ -108,7 +115,7 @@ private void init(Key key)
         }
         /* attempt to translate the key if it is not an ECKey */
         ECKey ecKey = ECKeyFactory.toECKey(key);
-        if (ecKey instanceof ECPrivateKeyImpl keyImpl) {
+        if (ecKey instanceof ECPrivateKey keyImpl) {
```

Comment:
The name `keyImpl` may be misleading as it isn't necessarily `ECPrivateKeyImpl` now.

Reply:
Changed.